sdk/python/setup.py

```diff
@@ -215,7 +210,7 @@ def run(self):
     # https://stackoverflow.com/questions/28509965/setuptools-development-requirements
     # Install dev requirements with: pip install -e .[dev]
     extras_require={
-        "dev": ["mypy-protobuf==1.*", "grpcio-testing==1.*"],
+        "dev": DEV_REQUIRED,
```

Comment:
nit: could you add `proto-plus` to `REQUIRED`? several folks have had [issues](https://github.com/feast-dev/feast/issues/2009) with this recently

Reply:
Done